lib/svg-sprite/config.js

```diff
@@ -212,33 +212,38 @@ module.exports = class SVGSpriterConfig {
      * @private
      */
     _getMetaData() {
-        if (!('meta' in this.shape && !isPlainObject(this.shape.meta))) {
+        if (!('meta' in this.shape) || isPlainObject(this.shape.meta)) {
```

Comment:
I think previous version was more obvious. New version can be shortened to  `if (isPlainObject(this.shape.meta)) because `isPlainObject(undefined) === false`